Sources/Components/ChristmasWishListView/ChristmasWishListView.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import Foundation
+
+public protocol ChristmasWishListViewDelegate: AnyObject {
+    func christmasWishListViewDidSelectReadMore(_ view: ChristmasWishListView)
```

Comment:
It's probably not useful in this case, but I think we should start adding sender view in these kinds of delegate methods, mainly for presenting popovers on iPad.

Reply:
I don't think we need a pop over in this case, so I would prefer to avoid adding more verbosity to the delegate methods until it's required

Reply:
Agree, wrote this as a reminder because it would be another breaking change in the public API if you actually needed pop overs, but forgot to send the source view.